pkg/sql/sem/builtins/pgcrypto_builtins.go

```diff
@@ -161,6 +184,148 @@ var pgcryptoBuiltins = map[string]builtinDefinition{
 	),
 }
 
+func crypt(password string, salt string) (string, error) {
+
+	var cryptFunc func([]byte, []byte) ([]byte, error)
+	if strings.HasPrefix(salt, "$1$") {
+		cryptFunc = cryptMd5
+	} else if strings.HasPrefix(salt, "$2a$") || strings.HasPrefix(salt, "$2x$") {
+		cryptFunc = cryptBlowfish
+	} else {
+		// todo des/xdes
```

Comment:
nit: comment usually of form `TODO(your name/issue number): some complete sentence`